### PR TITLE
Fix facebook share counter.

### PR DESCRIPTION
**Pull request by khustochka**
Fix for #152.

### CODE REVIEW

**Comment by khustochka** [comment]
Fix for #152.

File: source/services/facebook.js [hunk 1]
```diff
@@ -3,8 +3,8 @@
  */
 
 export default {
-    counterUrl: 'https://graph.facebook.com/?id={url}&callback=?',
-    convertNumber: (data) => data.share.share_count,
+    counterUrl: 'https://graph.facebook.com/?id={url}&fields=og_object%7Bengagement%7D&callback=?',
```
Comment (NikolayRys) [inline source/services/facebook.js]: Nice, but I would restrict the query only to the data which actually used:
`'https://graph.facebook.com/?id={url}&fields=og_object%7Bengagement%7Bcount%7D%7D&callback=?'`

Comment (khustochka) [reply source/services/facebook.js]: Thank you for the suggestion! I have updated the code.

**Comment by khustochka** [comment]
I have also upgraded chromedriver to make the tests run. There is one failure which is not related to  facebook counter but to Pinterest.

**Pull request merged**